bin/local-env/docker-compose.yml

```
@@ -44,4 +46,4 @@ services:
       MYSQL_DATABASE: wordpress_test
 
 volumes:
-  wordpress_data:
+  wp_data:
```

Why change this volume?

---

Ah that makes sense, although couldn't we delete it by running `docker-compose down -v` so that it's recreated next time?

Changing the volume name is a clever idea but maybe we should make the versioning a bit more explicit? 
```suggestion
  # Increment the volume name when the generate wp-config.php needs to be regenerated.
  wordpress_data_v2:
```

Alternatively, can we extract the extra constants into a file we mount so we don't need to recreate the volume just to add/change a constant? This could be the whole wp-config or just the extra parts defined in that env variable – then the `WORDPRESS_CONFIG_EXTRA` could simply be something like `require wp-config-extra.php;`?

---

> although couldn't we delete it by running `docker-compose down -v` so that it's recreated next time?

Yes, IMO this is the best option for us. I have updated the `env:stop` command to delete volumes when we stop the environment.